util/xsl/README.md

```diff
@@ -33,3 +33,23 @@ The --types option is provided to specify which types should be updated. It coul
 ``` bash
 $> --types <jobs/dyd/par/crv>
 ```
+
+## Use XSL script from a distribution
+
+XSL files can be used to update old test cases with a new dynawo distribution. Follow [this](https://github.com/dynawo/dynawo#using-a-distribution) on how to download a distribution.
+
+Then you can use the following commands to update a test case:
+
+``` bash
+$> unzip Dynawo_Linux_latest.zip
+$> cd dynawo/sbin/xsl
```

Comment:
I would prefer to wait until the dependency to nrt.py is fixed before advertising this, what do you think?

Reply:
It is a question I was not sure how to answer, I would still put it to make people aware of the possibility (and the folder is well hidden so I don't think a lot of people will see it anyway) and for us to know how to do

Reply:
People from the team should use the target into envDynawo.sh

Reply:
I'm just bothered that we expose a solution for which we have to apologize ;)